packages/internal-test-env/index.ts

```diff
@@ -78,100 +77,97 @@ export interface TestingEnvironmentBase {
 type FeatureFlags = {
   [key: string]: any;
 };
+const ENV_VAR_PREFIX = "E2E_TEST_FEATURE_";
+
 let envLoaded = false;
-let featuredFlags: FeatureFlags = {};
 export interface EnvVariables {
   // Common Envs
-  E2E_TEST_ENVIRONMENT: AvailableEnvironment;
+  E2E_TEST_ENVIRONMENT: AvailableEnvironments;
   E2E_TEST_IDP: string;
 }
 
 export function setupEnv() {
-  // If we're in CI, the environment is already configured.
-  if (process.env.CI) {
-    return;
-  }
-
-  if (envLoaded) {
+  // If we're in CI, the environment is already configured. Otherwise, it is
+  // loaded once.
+  if (process.env.CI || envLoaded) {
     return;
   }
-
   const envPath = join(process.cwd(), "e2e/env/.env.local");
 
-  // Otherwise load dotenv configuration
+  // Load dotenv configuration
   config({
     path: envPath,
   });
 
   if (!process.env.E2E_TEST_ENVIRONMENT) {
     console.error(
-      `We didn't find the given environment variable E2E_TEST_ENVIRONMENT,
-tried looking in the following directory for \`.env.local \`: 
-${envPath}`
+      `We didn't find the given environment variable E2E_TEST_ENVIRONMENT, tried looking in the following directory for '.env.local': ${envPath}`
```

Comment:
is this good with our line length via prettier?

Reply:
I ran prettier manually to check and it didn't complain